src/com/esotericsoftware/yamlbeans/WriteClassName.java

```diff
@@ -0,0 +1,5 @@
+package com.esotericsoftware.yamlbeans;
+
+public enum WriteClassName {
+	ALWAYS, NEVER, AUTO
```

Comment:
Oh, I see that `setAlwaysWriteClassname` was already a feature and you are just added never. Ok, an enum is fine but can you move it as a static member class for YamlConfig?